createmobilespec/createmobilespec.js

```diff
@@ -345,16 +345,13 @@ if (argv.global) {
 
     console.log("To clone repositories:");
     console.log(["  ." + path.sep + "cordova-coho" + path.sep + "coho repo-clone"].concat(repos).join(" -r "));
-    if (!argv.globalplugins) {
```

Comment:
what happened to this conditional

Reply:
I checked the commands being executed in there, they make no sense at all to me. They also really don't seem to be needed if you use `npm-link` as added below.